graphics/tinygl/clip.cpp

```diff
@@ -226,13 +226,12 @@ float(*clip_proc[6])(Vector4 *, Vector4 *, Vector4 *) =  {
 
 static inline void updateTmp(GLContext *c, GLVertex *q, GLVertex *p0, GLVertex *p1, float t) {
 	if (c->current_shade_model == TGL_SMOOTH) {
-		float a = q->color.W;
 		q->color = p0->color + (p1->color - p0->color) * t;
-		q->color.W = a;
```

Comment:
The alpha channel is `color.W`, right ?
`q`, `p0` and `p1` being GLVertex, `->color` is Vector4 and the operators it defines (`Vector4 + Vector4`, `Vector4 - Vector4` and `Vector4 * float`) all update the `W` component.